src/app/[locale]/kto/[answer]/[id]/components/KtoContainer.tsx

```
@@ -45,12 +45,12 @@ export const KtoContainer = ({ answer, id }: KtoContainerProps) => {
 
     async function load() {
       try {
-        const status = await getFeedbackStatus(id)
+        const status = await getFeedbackStatus(id, config.baseUrlApi)
 
         if (cancelled) return
 
         if (status.status === 'ok') {
-          const options = await getFeedbackOptions(isSatisfied)
+          const options = await getFeedbackOptions(isSatisfied, config.baseUrlApi)
```

ESLint

[error] Replace `isSatisfied,·config.baseUrlApi` with `⏎············isSatisfied,⏎············config.baseUrlApi⏎··········`

(prettier/prettier)